packages/docusaurus-remark-plugin-npm2yarn/src/index.js

```diff
@@ -57,7 +57,11 @@ const nodeForImport = {
 module.exports = (options = {}) => {
   const {sync = false} = options;
   let transformed = false;
+  let alreadyImported = false;
   const transformer = (node) => {
+    if (node.type === 'import' && node.value.includes('@theme/Tabs')) {
```

Comment:
I intentionally check import only one `Tabs` component, because I think that using `TabItem` without importing `Tabs` makes no sense at all, so we can omit checking these two components.